src/components/CoinCard.js

```diff
@@ -0,0 +1,107 @@
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import arrowUp from '../images/arrow-up.svg';
+import arrowDown from '../images/arrow-down.svg';
+
+CoinCard.propTypes = {
+  currency: PropTypes.string.isRequired,
+  rank: PropTypes.number.isRequired,
+  name: PropTypes.string,
+  symbol: PropTypes.string,
+  image: PropTypes.string,
+  price: PropTypes.number,
+  price_change_percentage_24h: PropTypes.number,
+};
+
+function CoinCard({
+  currency,
+  rank,
+  name,
+  symbol,
+  image,
+  price,
+  price_change_percentage_24h,
+}) {
+  return (
+    <CardWrapper>
+      <CoinRank>{rank}.</CoinRank>
+      <CoinImage>
+        <img alt={name} src={image} height="32" width="32"></img>
+      </CoinImage>
+      <CoinName>
+        <h3>{name}</h3>
+        <p>{symbol.toUpperCase()}</p>
+      </CoinName>
+      <CoinDetails>
+        <p>
+          {currency === 'eur'
+            ? `${price.toFixed(2)} €`
+            : `$${price.toFixed(2)}`}
+        </p>
+
+        {price_change_percentage_24h >= 0 ? (
+          <ColorGreen>
+            <img alt="Arrow up" src={arrowUp} height="12" width="12"></img>
+            {price_change_percentage_24h.toFixed(2)} %
+          </ColorGreen>
+        ) : (
+          <ColorRed>
+            <img alt="Arrow down" src={arrowDown} height="12" width="12"></img>
+            {price_change_percentage_24h.toFixed(2)} %
+          </ColorRed>
+        )}
+      </CoinDetails>
+    </CardWrapper>
+  );
+}
+
+export default CoinCard;
+
+const CardWrapper = styled.div`
```

Comment:
Same comment as above, maybe in their own file?

Reply:
Hey I did the styling with Styled components so that I can do in the same file :)